servicetalk-loadbalancer/docs/modules/ROOT/pages/defaultloadbalancer.adoc

```diff
@@ -1,21 +1,19 @@
-= DefaultLoadBalancer
+= ServiceTalks Default Load Balancer
 
-== What is DefaultLoadBalancer?
+== What is the Default Load Balancer?
```

Comment:
Overall everything LGTM. My only hesitation is that in our public API there is no "Default" in the name. So, it might be harder to relate docs to API. On the other hand, I don't have a better proposal other than making `Default` lowercase everywhere to highlight that this is our only default impl but not making it look like part of the name.

Would be nice to also link on this page to either an example page (https://apple.github.io/servicetalk/servicetalk-examples/SNAPSHOT/http/index.html#DefaultLoadBalancer) or `LoadBalancers` class or both.